src/signing/networkMessages/kusama.js

```diff
@@ -68,7 +72,8 @@ export async function UnstakeTx(senderAddress, { from, amount }, network) {
     transactions.push(await api.tx.staking.unbond(chainAmount))
   }
 
-  if (from.length > 0) {
+  // Disable adding possible nomination extrinsic if address is a stash account
+  if (from.length > 0 && addressRole !== `stash`) {
```

Comment:
```suggestion
  if (from.length > 0 && ["controller", "stash/controller", "none"].includes(addressRole)) {
```

inverted for clarity

Reply:
should include `undefined` as well no?

Reply:
ah no, because it is only for Kusama. Cool